### PR TITLE
Math magicians: Add unit testing

### PR DESCRIPTION
**Pull request by mateo951**
- Modified readme to fix link to issues page
- Added unit testing for all components
- Added linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ LinkedIn: [@Mateo Villagómez](https://www.linkedin.com/in/mateo-villagómez/)<b
 🤝 Contributing
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues](https://github.com/mateo951/to-do-list/issues) page.
+Feel free to check the [issues](https://github.com/mateo951/math-magicians/issues) page.
```
Comment (GzimAsani) [inline README.md:60]: - [x] Add the requirements that the user should install and the command that the user should use in order to test your project

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Required Changes :recycle: 

## Hello,   hope you're well :dove: You have done a great job so far :100:  

## To highlight:
- [ ] The gitflow has been followed :+1: 
- [ ] Great job with testing :+1: 

## To improve:
- [ ] Check inline comments for more information :thought_balloon:

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:

 Cheers and Happy coding!

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (mateo951) [reply README.md:60]: Done ✔

File: README.md [hunk 2]
```diff
@@ -13,16 +13,36 @@ Built With
...
+### Run tests
 
-#### To test the app locally, use Live Server (Extension for VS Code) to avoid js module CORS policy ####
+- Linters Tests:
+  - Eslint: run `npx eslint .`
+  - Stylelint: run `npx stylelint "**/*.{css,scss}"`
+- Run a Live Version of the Dist Folder:
+  - Run `npm start`
+- To Deploy to GH-Pages:
+  - Run `npm run deploy`
```
Comment (george-shammar) [inline README.md:45]: - [ ] The most important test command has been omitted: `npm run test` or `npm test`. Please let's add it. :pray:

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (mateo951) [reply README.md]: Done

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @mateo951, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**